src/port.js

```diff
@@ -183,7 +192,8 @@ export default class Port {
 					requestType,
 					{
 						id: id,
-						promise: resolve,
+						resolve,
+						reject
```

Comment:
Whoa... with the absence of a key, this is equivalent to: `{ id: id, resolve: resolve, reject: reject }`? Blind: blown.